.github/workflows/cd.yml

```diff
@@ -0,0 +1,69 @@
+name: CD
+
+on:
+  push:
+    tags:
+      - '*.*.*'
+
+jobs:
+  benchmark_tests:
+    name: benchmark tests
+    runs-on: ubuntu-latest
+    env:
+      SOURCE_PROJECT_KEY: java-sync-source
+      SOURCE_CLIENT_ID: ${{ secrets.SOURCE_CLIENT_ID }}
+      SOURCE_CLIENT_SECRET: ${{ secrets.SOURCE_CLIENT_SECRET }}
+      TARGET_PROJECT_KEY: java-sync-target
+      TARGET_CLIENT_ID: ${{ secrets.TARGET_CLIENT_ID }}
+      TARGET_CLIENT_SECRET: ${{ secrets.TARGET_CLIENT_SECRET }}
+      CI_BUILD_DIR: '/home/runner/work/commercetools-sync-java/commercetools-sync-java/'
+      GRGIT_USER: ${{ secrets.GRGIT_USER }}
+      GITHUB_ACTION_COMMIT: $GITHUB_SHA
```

Comment:
Looks like GITHUB_SHA does not works, according to benchmark reports :

see: https://commercetools.github.io/commercetools-sync-java/benchmarks/

<img width="242" alt="Screenshot 2021-02-02 at 19 22 49" src="https://user-images.githubusercontent.com/3469524/106644980-13fb5f80-658c-11eb-909c-f5dc38ab675f.png">

Benchmark commit:
https://github.com/commercetools/commercetools-sync-java/commit/dd3f96a5e510ca99d25d490094f7a5da5c98e0d5

Reply:
d66340fcb